docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.44 as builder
```

Comment:
It might be better to target the version `rust:1-buster` to get the latest stable version of Rust while ensuring we build on the same system as the exec image.

Reply:
done in 94f2bc4